com.microsoft.mrtk.core/Interactors/IVariableReticle.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// A reticle that implements some visual effect controllable by a single float value.
+    /// </summary>
+    public interface IVariableReticle
+    {
+        /// <summary>
+        /// Updates visuals as needed for the variable reticle.
+        /// </summary>
+        public void UpdateVisuals(VariableReticleArgs args);
+    }
+
+    public struct VariableReticleArgs
+    {
+        /// <summary>
+        /// XRRayInteractor that the reticle serves as a visual for.
+        /// </summary>
+        public XRRayInteractor RayInteractor;
+
+        /// <summary>
+        /// The reticle game object. 
+        /// </summary>
+        public GameObject Reticle;
```

Comment:
I'm pretty sure you don't need `Reticle`, please remove.  See comment in SpatialManipulateReticle for details